build.xml

```diff
@@ -66,18 +36,18 @@
     </target>
 
     <target name="compile-cubrid" depends="src-cubrid">
-        <javac destdir="${bin-cubrid}" source="1.5" target="1.5" encoding="EUC-KR" debug="true" debuglevel="lines,source,vars" deprecation="off" includeantruntime="no">
+        <javac destdir="${bin-cubrid}" source="1.6" target="1.6" encoding="EUC-KR" debug="true" debuglevel="lines,source,vars" deprecation="off" includeantruntime="no">
             <src path="${src-cubrid}"/>
             <include name="**/*.java"/>
         </javac>
     </target>
 
-    <target name="src-cubrid" depends="clean,init-jdk6">
+    <target name="src-cubrid" depends="clean">
```

Comment:
trivial: this target name is duplicated with the property name, `src-cubrid`. It may be confused for developers.

Reply:
I don't really like change for the minor problem. Because I could be wrong again and another verification follows.
But you are right. I will enjoy change.

Reply:
Yes, it is trivial so it is ok to ignore it. Anyway, thank you for considering my opinion :-)

Reply:
No, thank for your advice.